gc/stats/CollectionStatistics.hpp

```diff
@@ -45,6 +45,9 @@ class MM_CollectionStatistics : public MM_Base
 	uint64_t  _endTime;			/**< Collection end time */
 	uint64_t  _stallTime;		/**< Collection stall time */
 
+	uint64_t  _pauseTotal;		/**< Sum of all STW pauses during this GC cycle */
+	uint64_t  _pauseLongest;	/**< Longest STW pause during this GC cycle */
```

Comment:
Single spacing is sufficient. Please also correct lines 44-46 accordingly.
```suggestion
	uint64_t _pauseTotal;		/**< Sum of all STW pauses during this GC cycle */
	uint64_t _pauseLongest;	/**< Longest STW pause during this GC cycle */
```

Reply:
Fixed